cfme/tests/services/test_dialog_regex_validation_in_catalog.py

```diff
@@ -66,10 +72,23 @@ def test_dialog_element_regex_validation(appliance, dialog_cat_item):
     catalog_item, ele_name = dialog_cat_item
     service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
     view = navigate_to(service_catalogs, 'Order')
-    view.fields(ele_name).fill("!@#%&")
-    wait_for(lambda: view.submit_button.disabled, timeout=7)
-    view.fields(ele_name).fill("test_123")
-    wait_for(lambda: not view.submit_button.disabled, timeout=7)
+
+    for value in ["test_123", "!@#$%", ""]:
```

Comment:
This special character string is used three times, let's name it and use that name instead.